modules/siddhi-core/src/main/java/io/siddhi/core/table/InMemoryTable.java

```diff
@@ -213,6 +262,11 @@ public CompiledUpdateSet compileUpdateSet(UpdateSet updateSet, MatchingMetaInfoH
         return new InMemoryCompiledUpdateSet(expressionExecutorMap);
     }
 
+    @Override
+    protected void connectAndLoadCache() throws ConnectionUnavailableException {
+
+    }
+
     class TableState extends State {
 
         private final EventHolder eventHolder;
```

Comment:
Can't we improve event holder to monitor the number of events it has then calculating at the siddhi table level.